README.md

```diff
@@ -48,10 +48,9 @@ structures used.
 <a name="HUJIex"/>
 
 #### Loading data offline ####
```

Comment:
> they will take a longer time to startup. When developing you can uncomment the line
; addopts = --no-migrations (remove the ;) within Closure_Test/pytest.ini, to disable migrations and
make the test DB infer its schema from the model class metadata directly.

Please mention this on `README.md`, so future devs know it?

Reply:
Done